src/bot.py

```diff
@@ -0,0 +1,70 @@
+import time
+import random
+import tweepy
+from tweepy import api
+
+API_KEY = "cA5q5WNUmtF7S5GTVGRIT9TC0"
+API_SECRET_KEY = "yKVT9sV1pvKe8MgRWiw4L3c1C0ipVp3QVkkZp9CPBhZPcDrVIT"
+ACCESS_TOKEN = "1343829111479828480-xVZF3SwW8cNrbxUPqDvO02y3QPYHKX"
+ACCESS_SECRET_TOKEN = "JqOm8wX4uEPtvQimnouok66xFEte5sjDAqUc5JLA0K5EM"
```

Comment:
these keys should be saved in an .env file else anyone can access your bot 
I recommend you to change/revoke these keys ASAP

Reply:
Okayy